content/200-orm/050-overview/500-databases/300-postgresql.mdx

```diff
@@ -29,6 +29,64 @@ The fields passed to the `datasource` block are:
 - `provider`: Specifies the `postgresql` data source connector.
 - `url`: Specifies the [connection URL](#connection-url) for the PostgreSQL database server. In this case, an [environment variable is used](/orm/prisma-schema/overview#accessing-environment-variables-from-the-schema) to provide the connection URL.
 
+## Using the <inlinecode>node-postgres</inlinecode> driver
+
+As of [`v5.4.0`](https://github.com/prisma/prisma/releases/tag/5.4.0), you can use Prisma ORM with database drivers from the JavaScript ecosystem (instead of using Prisma ORM's built-in drivers.) You can do this by using a [driver adapter](/orm/overview/databases/database-drivers).
```

Comment:
Probably a style question:
```suggestion
As of [`v5.4.0`](https://github.com/prisma/prisma/releases/tag/5.4.0), you can use Prisma ORM with database drivers from the JavaScript ecosystem (instead of using Prisma ORM's built-in drivers). You can do this by using a [driver adapter](/orm/overview/databases/database-drivers).
```